NEWS.md

```diff
@@ -1,4 +1,18 @@
-## 2.9.0 - Unreleased
+## 2.10.0 - Unreleased
```

Comment:
mod-login now has the latest version 7.3
https://github.com/folio-org/mod-login/blob/master/descriptors/ModuleDescriptor-template.json#L7
pleas use it in module-descriptor-template

Reply:
gobi version should be 1.12 instead of 1.11
https://github.com/folio-org/mod-gobi/blob/master/descriptors/ModuleDescriptor-template.json#L7

Reply:
Updated

Reply:
and what about gobi version?